package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "dependencies": {
+    "lodash": "^4.17.21",
+    "node": "^16.3.0"
+  },
+  "name": "finaltodoapp",
+  "version": "1.0.0",
+  "main": "todo_final.js",
+  "devDependencies": {
+    "@types/jest": "^26.0.24",
+    "jest": "^27.0.6",
+    "prettier": "^2.3.2",
+    "typescript": "^4.3.5"
+  },
```

Comment:
`ts-jest` needs to be installed as a dev dependency for the tests to run properly. re-install all packages using `npm ci` and then run tests to make sure they work.